src/Pact/Types/Native.hs

```diff
@@ -43,10 +43,10 @@ isSpecialForm = (`M.lookup` sfLookup)
 
 -- | Native function with un-reduced arguments that computes gas.
 --   Operation cost implemented in definition.
-type NativeFun e = FunApp -> [Term Ref] -> Eval e (Gas,Term Name)
+type NativeFun e = FunApp -> [Term Ref] -> Eval e (Term Name)
 
 -- | Native function with reduced arguments, initial gas pre-compute that computes final gas.
-type GasRNativeFun e = Gas -> FunApp -> [Term Name] -> Eval e (Gas,Term Name)
+type GasRNativeFun e = FunApp -> [Term Name] -> Eval e (Term Name)
```

Comment:
Potentially delete alias now that we don't thread gas through.

Reply:
I would love to see the alias dropped.  Big change - maybe in a followup PR?